standard_plus_issue/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_standard_plus_issue,access_standard_plus_issue,model_standard_plus_issue,,1,1,1,1
-access_issue_screenshot,access_issue_screenshot,model_issue_screenshot,,1,1,1,1
+access_standard_plus_issue_user,access_standard_plus_issue_user,model_standard_plus_issue,group_issue_user,1,1,1,0
+access_issue_screenshot_user,access_issue_screenshot_user,model_issue_screenshot,group_issue_user,1,1,1,0
```

Comment:
Is that ok that the user is now unable to delete his own screenshots?

Reply:
No it's not ! I'll fix it !